pages/records/get-veteran-id-cards/apply.md

```diff
@@ -1,13 +0,0 @@
----
-title: Veteran ID Card
-layout: page-react.html
-entryname: veteran-id-card
-hideFromSidebar: true
----
-<nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs"
-id="va-breadcrumbs">
-  <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
-    <li><a href="/">Home</a></li>
-    <li><a aria-current="page" href="/records/get-veteran-id-cards/apply/">Apply for an ID Card</a></li>
-  </ul>
-</nav>
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-85e913b0f71fcdb04b1901eeaafc2231R5. Breadcrumbs look good.